testing/code/test_source.py

```diff
@@ -96,28 +99,30 @@ def test_source_putaround():
     assert str(source).strip() == "if 1:\n    x=1"
 
 
-def test_source_strips():
+def test_source_strips() -> None:
     source = Source("")
     assert source == Source()
     assert str(source) == ""
     assert source.strip() == source
 
 
-def test_source_strip_multiline():
+def test_source_strip_multiline() -> None:
     source = Source()
     source.lines = ["", " hello", "  "]
     source2 = source.strip()
     assert source2.lines == [" hello"]
 
 
-def test_syntaxerror_rerepresentation():
+def test_syntaxerror_rerepresentation() -> None:
     ex = pytest.raises(SyntaxError, _pytest._code.compile, "xyz xyz")
+    assert ex is not None
     assert ex.value.lineno == 1
     assert ex.value.offset in {5, 7}  # cpython: 7, pypy3.6 7.1.1: 5
+    assert ex.value.text is not None
     assert ex.value.text.strip(), "x x"
```

Comment:
Could check the full expected text instead?
`assert ex.value.text == "xyz xyz\n"`

Reply:
Yes, that's better.